core/src/main/java/com/datastax/oss/driver/api/core/metadata/Node.java

```diff
@@ -149,4 +155,21 @@
    */
   @Nullable
   UUID getSchemaVersion();
+
+  /**
+   * The last time that the driver received a response from this node, or -1 if it has not responded
+   * yet.
+   *
+   * <p>This is recorded with {@link System#nanoTime()}, and should only be used to measure elapsed
+   * time.
+   *
+   * <p>Note that this feature is controlled by a configuration option ({@code
+   * advanced.metadata.nodes.last-response-time.enabled}) and <b>disabled by default</b> (i.e. will
+   * always return -1). You might want to enable it for information purposes, or for use in custom
+   * policy implementations. It incurs a bit of overhead (namely, a call to {@code
+   * System.nanoTime()} and a volatile write for every request).
```

Comment:
It looks like in `CqlRequestHandlerBase` it was already calling `System.nanoTime()` for the purpose of metrics, but I think the documentation is fine since that detail is impl specific.

Reply:
Right, I wrote that before the implementation. `CqlRequestHandlerBase` should generally represent the majority of the traffic, so I'll just mention the volatile write.

On a side note, the `nanoTime` call is not needed at all if both this feature and metrics are disabled. We could try to handle that a bit better, I'll open a separate ticket.